doc/source/building/linux.rst

```diff
@@ -1,75 +1,27 @@
-#############################
-Building from source on Linux
-#############################
-
-====================
-Generic instructions
-====================
-
-To build NumPy/SciPy from source, get the `source package
-<https://github.com/scipy/scipy>`__, unpack it, and:
-
-::
-
-   python setup.py install --user   # installs to your home directory
-
-or
-
-::
-
-   python setup.py build
-   python setup.py install --prefix=$HOME/local
-
-Before building, you will also need to install packages that NumPy and
-SciPy depend on
-
-* BLAS and LAPACK libraries (optional but strongly recommended for
-  NumPy, required for SciPy): typically `ATLAS
-  <http://math-atlas.sourceforge.net/>`__ + `OpenBLAS
-  <https://github.com/xianyi/OpenBLAS/>`__, or `MKL
-  <https://software.intel.com/en-us/intel-mkl>`__.
-
-* C, C++, and Fortran compilers (typically ``gcc``, ``g++``, and ``gfortran``).
-
-* Python header files (typically a package named ``python3-dev`` or ``python3-devel``)
-
-Typically, you will want to install all of the above from packages
-supplied by your Linux distribution, as building them yourself is
-complicated. If you need to use specific BLAS/LAPACK libraries, you
-can do
+.. _build-linux:
 
-::
-
-   export BLAS=/path/to/libblas.so
-   export LAPACK=/path/to/liblapack.so
-   export ATLAS=/path/to/libatlas.so
-   python setup.py ............
-
-* The `Cython <https://cython.org/>`__ and `Pythran <https://pythran.readthedocs.io>`__
-  ahead-of-time compilers are also necessary, as is ``pybind11``. It is
-  recommended to install these packages with ``pip`` or ``conda``, because it
-  is possible (even likely) that you need newer versions of these packages than
-  the ones that are available in your Linux distribution.
-
-
-Below, you will find additional installation instructions and advice
-for many major Linux distributions.
+Building from source on Linux
+=============================
 
+Debian/Ubuntu Linux
```

Comment:
If we're going to keep this section, how about using a tab for the different main Linux distros (Debian/Ubuntu, Fedora, CentOS/RHEL, Arch) to clean this up?

Reply:
Is this too much? I'm not sure I like the tabs within tabs but maybe that's ok.